### PR TITLE
Add confirmation dialog before resetting to default settings

### PR DESCRIPTION
**Pull request by storvik**
This pull request adds a simple confirmation dialog before resetting settings to default. I kept pressing the reset button accidentally. 

Thanks for this project, love the idea!

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt [hunk 1]
```diff
@@ -750,6 +736,54 @@ fun LookAndFeelActivity(
                     onValueChange = { text = it },
                     label = { Text(stringResource(R.string.test_out_thumbkey)) },
                 )
+                if (confirmReset.value) {
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt:739]: Move this entire block above, into the correct reset onClick. That onclick should open this dialog.

**Review by dessalines — COMMENTED** [review]
This probably isn't necessary, but I spose I'm okay with it.

Run `./gradlew formatKotlin` to pass lint.

File: app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt (hunk 1 above)
Comment (storvik) [reply app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt:739]: Seems like I cannot move it inside the onClick as this results in an error: 
`@Composable invocations can only happen from the context of a @Composable function`.

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt:739]: Ah. In that case, name the variable like `showConfirm...` and move it to the top level.

**Comment by storvik** [comment]
@dessalines should be fixed now

File: app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt [hunk 2]
```diff
@@ -750,6 +736,56 @@ fun LookAndFeelActivity(
                     onValueChange = { text = it },
                     label = { Text(stringResource(R.string.test_out_thumbkey)) },
                 )
+                if (showConfirmResetDialog.value) {
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt:739]: Why is this inside of a random outlined text field, it should be at the top level.

Comment (storvik) [reply app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt]: Ahh, think I misunderstood what you were saying.

**Review by dessalines — APPROVED** [review]
Sweet, thx!

**Pull request merged**